### PR TITLE
Flash error message for 4 seconds

### LINKED ISSUE #6: add/remove失败时显示原因
add已有名字或者remove不存在名字时弹窗告诉用户原因

### LINKED ISSUE #10: Improve UI

### PR DESCRIPTION
**Pull request by philip-zhan**
localhost_8080_groups_5137483062837248(iPhone X)

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -132,60 +134,66 @@ def process_players(players: List, limit: int) -> Tuple[List, List]:
...
+            return None
 
 
-def process_remove(group_id: str, player_type: str, player_name: str):
+def process_remove(group_id: str, player_type: str, player_name: str) -> Optional[str]:
     with datastore_client.transaction() as transaction:
         group_entity = fetch_group(group_id, transaction=transaction)
         if not group_entity:
-            return
+            return f"Can't find group with ID {group_id}"
```
Comment (t3rryw1) [inline main.py:196]: Come to think of it, feels like `try` and `except` would be better in this method...

File: main.py [hunk 2]
```diff
@@ -132,60 +134,66 @@ def process_players(players: List, limit: int) -> Tuple[List, List]:
...
         else:
             group_entity[player_list_name] = new_players
             datastore_client.put(group_entity)
+            return None
 
 
 @app.route("/", methods=["GET"])
 def root():
-    groups = fetch_groups()
+    groups = fetch_groups(10)
```
Comment (t3rryw1) [inline main.py]: add a constant `PAGE_LIMIT=10`?

**Comment by t3rryw1** [comment]
Overall, good job adding errors processing and formatting code!!!

File: main.py (hunk 1 above)
Comment (philip-zhan) [reply main.py:196]: Wrapped all db calls with try/except in the last commit.

**Pull request merged**